.github/workflows/build_and_test.yml

```diff
@@ -393,9 +397,11 @@ jobs:
 
   sparkr:
     needs: [configure-jobs, precondition]
+    # Run scheduled jobs only with JDK 17
+    # Run regular jobs only if sparkr changes exist
```

Comment:
better also mentioned for `Apache Spark only` and `for commit in both Apache Spark and forked repository`